lib/internal/process/warning.js

```diff
@@ -46,4 +46,10 @@ function setupProcessWarnings() {
     }
     process.nextTick(() => process.emit('warning', warning));
   };
+
+  process.emitExperimentalWarning = function(feature) {
+    const msg = `${feature} is an experimental feature. ` +
+                'Implementation details may change.';
```

Comment:
I think the wording should be improved... the exposed feature *could* change at any time.